packages/postcss-reduce-initial/data/toInitial.json

```diff
@@ -17,17 +17,19 @@
   "box-sizing": "content-box",
   "column-rule-color": "currentcolor",
   "font-synthesis": "weight style",
+  "image-orientation": "from-image",
   "mask-clip": "border-box",
   "mask-mode": "match-source",
   "mask-origin": "border-box",
-  "mask-repeat": "repeat",
+  "mask-repeat": "no-repeat",
   "mask-type": "luminance",
   "ruby-align": "space-around",
   "ruby-merge": "separate",
+  "ruby-position": "alternate",
   "text-decoration-color": "currentcolor",
   "text-emphasis-color": "currentcolor",
   "text-emphasis-position": "over right",
-  "transform-box": "border-box",
+  "transform-box": "view-box",
```

Comment:
It can be problem... I remember some browsers have wrong `mask-repeat` and `transform-box` values, so we remove them, but we should left a comment... maybe make sense too look at git history to check it, a lot of time has passed

Reply:
mask-repeat has already been manually changed to `repeat` in the past https://github.com/cssnano/cssnano/commit/cebd6d5b789a9c700f92dc8b40cdd8ef9545441a#diff-a3304505493004c674a19e13da735a9ea783c7f76e93c67ad0bb14334a67225e So I think we should probably follow the spec and not Mozilla. Maybe add a test like https://github.com/cssnano/cssnano/pull/852/files to avoid repeating the error?

Reply:
Yes, let's add test and comments in tests